src/test/java/uk/gov/justice/digital/config/ResourceLoader.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.digital.config;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.apache.commons.io.IOUtils;
+
+public class ResourceLoader {
+	
+	@SuppressWarnings("deprecation")
+	public static String getResource(final Class<?> clazz, final String resource) throws IOException {
```

Comment:
Following on from my comment above, consider replacing this with the code that reads resources in the `SourceReferenceService`. It works and is much simpler.

Also remove any duplicate code that handles reading resources for the tests since we should only have to declare it once.